### ISSUE TITLE
Rails 6 support

### ISSUE THREAD

**Issue by mrageh** [rails/activerecord-session_store#147]
Is this gem going to support Rails 6? 

At the moment I get this error when I attempt to upgrade an app that uses this gem to Rails 6.

```
Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    activerecord-session_store (~> 1.1.3) was resolved to 1.1.3, which depends on
      railties (>= 4.0)
```

**Comment by mrageh** [comment]
My bad @rafaelfranca and thanks for the swift response 😊

**Comment by rafaelfranca** [comment]
Given you thought it was this gem the cause did you actually find the cause? It may be something we should look at

**Comment by mrageh** [comment]
One of our internal dependencies is tied to an older version of `sass-rails`and I believe that's what's causing the problem. 

In order to fix it, we have to upgrade the dependecy to use the latest version of sass-rails (which I believe supports Rails 6). Before I can go ahead and continue upgrading Rails.
```    
sass-rails (~> 5.0.7) was resolved to 5.0.7, which depends on
railties (>= 4.0.0, < 6)
```
